meson.build

```diff
@@ -90,6 +95,8 @@ subdir('src')
 subdir('plugins')
 
 install_subdir('shaders', install_dir: 'share/wayfire')
+install_data('wayfire.ini.default', rename: 'wayfire.ini',
+               install_dir: join_paths(sysconfdir, 'wayfire'))
```

Comment:
I don't think we should install the default config file. This would require installation as root. Ideally, wayfire defaults should be enough to get things up and running without a config file. Perhaps wf-panel and wf-background could be ran by default if no config file is found, for example. Perhaps a choice background image could be used by default. The rest of the options already have hard-coded defaults.

The logic of looking for the config file in sysconfigdir as a fallback is fine, I just don't think the file should be installed by default. Perhaps another option could be added that installs the config file but is disabled by default.

Reply:
The default paths used for installation already require privileges. But I like idea with hard-coded defaults instead of default config installation but it could be undesirable for some use-cases.

Reply:
I like this idea, and see no problem with installing a default file, many projects do that. The only thing I want is that if the user specifies a nonstandard prefix, then the config file is installed to that prefix, thus requiring no root access.

Reply:
> The only thing I want is that if the user specifies a nonstandard prefix, then the config file is installed to that prefix, thus requiring no root access.
It already works that way. I build rpm package using that.

Reply:
If the config file is to be installed, it probably should be processed by meson so the default paths hard coded to /usr in the config file are changed to whatever prefix wayfire is installed with, and perhaps rename wayfire.ini.default to wayfire.ini.in. I think the only two places are in [autostart] section for wf-panel and wf-background.

Reply:
@wladmis So then it is ok I think. But it would be cool if you could add what @soreau described.

Reply:
I would try $XDG_CONFIG_DIR, $HOME/.config, /etc/wayfire and then /usr/share/wayfire paths order to find the config.

Reply:
/usr/share/wayfire??